Infrastructure/SetEnvironmentSecretsTemplate.sh

```diff
@@ -12,13 +12,21 @@
 # Name - This is the bit after 'sea-alpha-' - e.g. for sea-alpha-DEV, PAAS_ENV_SHORTNAME would just be 'DEV'
 PAAS_ENV_SHORTNAME="<REAL_VALUE_HERE>"
 
+# Basic Auth credentials to access the site
+cf set-env "sea-alpha-${PAAS_ENV_SHORTNAME}" BasicAuthUsername "<REAL_VALUE_HERE>"
+cf set-env "sea-alpha-${PAAS_ENV_SHORTNAME}" BasicAuthPassword "<REAL_VALUE_HERE>"
+
```

Comment:
At some point we won't need this anymore right? (But we probably need it for now)

Reply:
Not sure, we might keep it, it's already disabled in production